src/_gettsim_tests/test_rounding.py

```diff
@@ -131,6 +144,40 @@ def test_func(income):
     np.array_equal(calc_result["test_func"].values, np.array(input_values_exp_output))
 
 
+@pytest.mark.parametrize(
+    "base, direction, add, input_values, exp_output",
+    rounding_specs_and_exp_results_with_additive_part,
+)
+def test_rounding_with_additive_part(base, direction, add, input_values, exp_output):
```

Comment:
I don't think we need a separate test from `test_rounding` anymore, unless we leave the code as is.

Reply:
I removed the `test_rounding_with_additive_part` and included the test in the existing machinery. I wanted to explicitly test the case where `to_add_after_rounding` is not specified in the dict, so I edited the part where this is relevant in the original test.